src/containers/pages/InterventionPlan/NewPlan/General/index.tsx

```diff
@@ -73,6 +73,17 @@ const planFormPropsLookUp = {
     jurisdictionLabel: FOCUS_AREA_HEADER,
     redirectAfterAction: PLAN_LIST_URL,
   },
+  [InterventionType.IRSLite]: {
+    allowMoreJurisdictions: true,
+    cascadingSelect: true,
+    initialValues: {
+      ...defaultInitialValues,
+      activities: planActivitiesMap[InterventionType.IRS],
+      interventionType: InterventionType.IRS,
+    },
+    jurisdictionLabel: FOCUS_AREA_HEADER,
+    redirectAfterAction: PLAN_LIST_URL,
+  },
```

Comment:
@justinlewis @jsmethie let's please revert this one as well.  We are not yet adding IRSLite to `NewPlan`

Reply:
Done